### PR TITLE
[ACA-4041] [ADW] Sort order doesn't work correctly after opening a document

### PR DESCRIPTION
**Pull request by dhrn**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [x] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

https://issues.alfresco.com/jira/browse/ACA-4041

**What is the new behaviour?**

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Review by davidcanonieto — APPROVED** [review]

**Review by popovicsandras — APPROVED** [review]

File: src/app/components/files/files.component.html [hunk 1]
```diff
@@ -30,7 +30,6 @@
           [sorting]="['name', 'ASC']"
           [imageResolver]="imageResolver"
           [headerFilters]="true"
-          [filterValue]="queryParams"
```
Comment (mauriziovitale) [inline src/app/components/files/files.component.html:33]: why ?

Comment (mauriziovitale) [reply src/app/components/files/files.component.html:33]: in addition there is no unit test

Comment (mauriziovitale) [reply src/app/components/files/files.component.html:33]: @davidcanonieto I saw that was introduced by you. Can you double-check?
https://github.com/Alfresco/alfresco-content-app/commit/51184e5b16433ce6e93c73e28d97775f0b3e2b5e#diff-bda27cbfa7ca0c26a8b8679e0e5ed023R33

File: src/app/components/files/files.component.spec.ts [hunk 2]
```diff
@@ -239,6 +239,13 @@ describe('FilesComponent', () => {
       expect(router.navigate).toHaveBeenCalledWith(['personal-files', 'favourites', node.id]);
     });
 
+    // it('should navigates to node when is more that one sub node', () => {
+    //   router.url = '/personal-files?name=abc';
+    //   component.navigate(node.id);
+    //
+    //   expect(router.navigate).toHaveBeenCalledWith(['personal-files', node.id]);
+    // });
```
Comment (davidcanonieto) [inline src/app/components/files/files.component.spec.ts:247]: why is it commented?

File: src/app/components/files/files.component.html (hunk 1 above)
Comment (davidcanonieto) [reply src/app/components/files/files.component.html:33]: we should not remove this property, it enables the header filters

Comment (davidcanonieto) [reply src/app/components/files/files.component.html:33]: Never mind, it's added underneath

File: src/app/components/files/files.component.spec.ts (hunk 2 above)
Comment (dhrn) [reply src/app/components/files/files.component.spec.ts:247]: failing now. i am fixing it

**Review by eromano — APPROVED** [review]

**Review by davidcanonieto — APPROVED** [review]

**Review by davidcanonieto — APPROVED** [review]

**Pull request merged**